rspec-mode.el

```diff
@@ -977,7 +977,9 @@ or a cons (FILE . LINE), to run one example."
 (defun rspec--autosave-buffer-maybe ()
   "Saves the current buffer if `rspec-autosave-buffer' is t and
 the buffer is a spec or a target file."
-  (when (and rspec-autosave-buffer (rspec-spec-or-target))
+  (when (and rspec-autosave-buffer
+             (or (rspec-buffer-is-spec-p)
```

Comment:
Should we just remove the rspc-spec-or-target check instead? After all, unsaved other files can affect the running of the specs just as much.

Reply:
I'd be fine with that. I ended up replacing this function with a save all anyway, since that's usually what I want to do.

Reply:
Updated